packages/components/src/templates/next/components/internal/Navbar/Navbar.tsx

```diff
@@ -67,21 +67,122 @@ export const Navbar = ({
   }, [])
 
   return (
-    <div className="flex flex-col">
+    <div className="relative flex flex-col">
       {/* Site header */}
       <div
-        className="mx-auto flex w-full max-w-container flex-row px-6 py-6 lg:px-10"
+        className="max-w-container mx-auto flex w-full flex-row gap-4 px-6 py-6 lg:px-10"
         ref={siteHeaderRef}
       >
         {/* Logo */}
-        <LinkComponent href="/">
+        <LinkComponent href="/" className="my-auto">
           <img
             src={logoUrl}
             alt={logoAlt}
-            className="h-11 w-full max-w-[110px] object-contain object-center lg:h-20 lg:max-w-[200px]"
+            className="h-11 w-32 max-w-[110px] object-contain object-center lg:h-12 lg:max-w-[128px]"
           />
         </LinkComponent>
 
+        {/* Navigation items (for desktop) */}
+        <div className="mx-auto hidden w-full content-center lg:block">
+          <ul
+            className="mt-2 flex flex-row flex-wrap gap-1"
+            ref={navDesktopRef}
+          >
+            {items.map(({ name, url, description, items }, idx) => {
+              if (!items || items.length === 0) {
+                return (
+                  <li key={`${name}-${idx}`}>
+                    <LinkComponent
+                      className="block px-2 py-1 text-base/5 font-medium text-[#1f2937]"
+                      href={url}
+                    >
+                      {name}
+                    </LinkComponent>
+                  </li>
+                )
+              }
```

Comment:
shouldn't this be done outside of the `map`? if `items === undefined`, we have an error here ._.

Reply:
Lol shucks variable scoping issue, renamed in 9b3752599270489aa3c949a809e2d2156d54cca3 to be clear what the `items` here mean.